app/internal/timer.py

```diff
@@ -0,0 +1,21 @@
+from app.routers.event import sort_by_date
+from app.routers.user import get_all_user_events
+from sqlalchemy.orm import Session
+from app.database.models import Event
+
+
+def get_next_user_event(session: Session, user_id: int) -> Event:
+    events_as_list = list(sort_by_date(get_all_user_events(session, user_id)))
+    next_event = None
+    if len(events_as_list) > 0:
+        next_event = events_as_list[0]
```

Comment:
You can just return it here, and spare the next line and the assignment two lines above

Reply:
Done